boards/arm/pinetime_devkit0/pinetime_devkit0.dts

```diff
@@ -147,6 +147,30 @@
 		label = "XT25FB32";
 		jedec-id = [0b 40 16];
 		size = <DT_SIZE_M(32)>;
+
+		partitions {
+			compatible = "fixed-partitions";
+			#address-cells = <1>;
+			#size-cells = <1>;
+
+			/* next firmware update */
+			slot1_partition: partition@0 {
+				label = "image-1";
+				reg = <0x00000000 0x74000>;
+			};
+
+			/* needed to slot0-slot1 swapping */
+			scratch_partition: partition@74000 {
+				label = "image-scratch";
+				reg = <0x00074000 0xa000>;
+			};
```

Comment:
No a good idea because swapping happens on unencrypted chunks, this means that during swap, by tapping to spi lines, it is possible to get complete unencrypted image of an application.

Reply:
umcomplete unencrypted image?

Reply:
https://github.com/mcu-tools/mcuboot/blob/main/docs/design.md#swap-without-using-scratch